semeio/workflows/localisation/local_script_lib.py

```diff
@@ -21,6 +20,7 @@
 
 # The global variable is only used for controlling output of log info to screen
 debug_level = log_level_setting.debug_level
+scaling_param_number = log_level_setting.scaling_param_number
```

Comment:
I really think we should avoid global parameters, so if you have a rewrite that will get rid of it, I suggest you add that in this PR.

Reply:
This is fixed and there are no longer any global parameter.